modules/persistent-storage-csi-ebs-operator-uninstall.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+
+[id="persistent-storage-csi-ebs-operator-uninstall_{context}"]
+= Uninstalling the AWS Elastic Block Store CSI Driver Operator
+
+Before you uninstall the AWS EBS CSI Driver Operator, you must delete all persistent volume claims (PVCs) that are in use by the Operator.
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+.Procedure
+To uninstall the AWS EBS CSI Driver Operator from the web console:
+
+. Log in to the web console.
+
+. Navigate to *Storage* -> *Persistent Volume Claims*.
+
+. Select any PVCs that are in use by the AWS EBS CSI Driver Operator and click *Delete*.
+
+. From the *Operators* -> *Installed Operators* page, scroll or type *AWS EBS CSI* into the *Filter by name* field to find the Operator. Then, click on it.
+
+. On the right-hand side of the *Operator Details* page, select *Uninstall Operator* from the *Actions* drop-down menu.
+
+. When prompted by the *Remove Operator Subscription* window, you can select the *Also completely remove the Operator from the selected namespace* check box to remove all components related to the Operator. This removes the CSV, which in turn removes the Pods, Deployments, CRDs, and CRs associated with the Operator. Leaving this unchecked results in only the Subscription being removed.
+
+Select *Remove*. This Operator will stop running and no longer receive updates.
```

Comment:
Instructions are based on https://docs.openshift.com/container-platform/4.4/operators/olm-deleting-operators-from-cluster.html. Need to test.